src/main/java/org/broadinstitute/consent/http/authentication/OAuthAuthenticator.java

```diff
@@ -28,8 +30,7 @@ public OAuthAuthenticator(SamService samService) {
   }
 
   @Override
-  public Optional<AuthUser> authenticate(String bearer) {
-    try {
+  public Optional<AuthUser> authenticate(String bearer) throws AuthenticationException {
```

Comment:
Minor nit ... after removing the try/catch, the inner code is no longer indented correctly. Try reformatting the code to apply the project code style.